docs/pip/compile.md

```diff
@@ -60,6 +60,18 @@ $ uv pip compile pyproject.toml --all-extras
 
 Note extras are not supported with the `requirements.in` format.
 
+To lock dependency groups in the current project directory's pyproject.toml:
+
+```console
+$ uv pip compile --group foo
+```
+
+To lock dependency groups for an arbitrary pyproject.toml:
+
+```console
+$ uv pip compile --group some/path/pyproject.toml:foo
+```
+
```

Comment:
We should also mention `--project` here, especially for multiple groups.

Reply:
We may want to add a !!! note admonition noting that this is not yet implemented in `pip-tools` yet and link to the issue?